packages/components/src/workers/parquetWorkerClient.ts

```diff
@@ -1,4 +1,5 @@
 import { asyncBufferFromUrl, cachedAsyncBuffer, AsyncBuffer, ParquetReadOptions, FileMetaData } from 'hyparquet'
+import ParquetWorker from './parquetWorker?worker'
```

Comment:
haven't seen this syntax before. what does ?worker do?

Reply:
It's detailed here: https://vite.dev/guide/features.html#import-with-query-suffixes. It does not seem to be the recommended way though.

Reply:
fixed.

Reply:
see https://github.com/hyparam/hyperparam-cli/pull/14#discussion_r1829193962, where I use it again.